lib/galaxy/managers/jobs.py

```diff
@@ -380,7 +382,7 @@ def by_tool_input(
         tool_version: Optional[str],
         param: ToolStateJobInstancePopulatedT,
         param_dump: ToolStateDumpedToJsonInternalT,
-        job_state: Optional[JobStatesT] = "ok",
+        job_state: Optional[Union[JobStatesT, JobStatesT]] = (Job.states.OK, Job.states.SKIPPED),
```

Comment:
This `Union` seems redundant or wrong?

Reply:
Oops, yeah, I think that went wrong when switching branches.
```suggestion
        job_state: Optional[JobStatesT] = (Job.states.OK, Job.states.SKIPPED),
```